package.json

```
@@ -64,7 +64,7 @@
     "prettier": "1.18.2",
     "semantic-release": "^15.13.24",
     "ts-jest": "^25.3.0",
-    "typescript": "^3.8.3"
+    "typescript": "^4.0.3"
   },
   "peerDependencies": {
     "eslint": "^5 || ^6"
```

This should be updated to greater or equal than 5.
```suggestion
    "eslint": ">=5.0.0"
```